home/tests/test_broken_links_clean_up.py

```diff
@@ -0,0 +1,130 @@
+from io import StringIO
+
+from django.core.management import call_command  # type: ignore
+from django.test import TestCase  # type: ignore
+from wagtail.models import Page  # type: ignore
+
+from home.models import HomePage
+
+from .page_builder import (
+    NextBtn,
+    PageBtn,
+    PageBuilder,
+    WABlk,
+    WABody,
+)
+
+
+class TestBrokenLinks(TestCase):
+    def setUp(self) -> None:
+        """
+        Create a page with no related pages, the `related_page`
+        """
+        home_page = HomePage.objects.first()
+        self.main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
+        self.ha_menu = PageBuilder.build_cpi(
+            self.main_menu, "ha-menu", "HealthAlert menu"
+        )
+        self.health_info = PageBuilder.build_cp(
+            parent=self.ha_menu,
+            slug="health-info",
+            title="health info",
+            bodies=[WABody("health info", [WABlk("*Health information*")])],
+        )
+
+    def test_content_page_with_related_pages(self) -> None:
+        """ """
+        output = StringIO()
+
+        self_help = PageBuilder.build_cp(
+            parent=self.ha_menu,
+            slug="self-help",
+            title="self-help",
+            bodies=[WABody("self-help", [WABlk("*Self-help programs*")])],
+        )
+        index = PageBuilder.build_cpi(self.health_info, "iddex-page", "Index Page")
+
+        self_help_rp = Page.objects.get(pk=self_help.id)
+        index_rp = Page.objects.get(pk=index.id)
+        PageBuilder.link_related(self.health_info, [self_help, index])
+
+        call_command("broken_links_clean_up", stdout=output)
+        assert len(self.health_info.related_pages) == 2
+        assert len([rp.value for rp in self.health_info.related_pages]) == 2
+        assert [rp.value for rp in self.health_info.related_pages] == [
+            self_help_rp,
+            index_rp,
+        ]
+        assert output.getvalue().strip() == "Successfully retrieve broken links"
+
+    def test_content_page_with_deleted_related_page(self) -> None:
+        """
+        If related_page is deleted we still get two objects but one will be null the one that is deleted
+        """
+        output = StringIO()
+
+        self_help = PageBuilder.build_cp(
+            parent=self.ha_menu,
+            slug="self-help",
+            title="self-help",
+            bodies=[WABody("self-help", [WABlk("*Self-help programs*")])],
+        )
+        index = PageBuilder.build_cpi(self.health_info, "iddex-page", "Index Page")
+
+        self_help_rp = Page.objects.get(pk=self_help.id)
+        PageBuilder.link_related(self.health_info, [self_help, index])
+
+        index.delete()
+        related_pages = [rp.value for rp in self.health_info.related_pages]
+        call_command("broken_links_clean_up", stdout=output)
+
+        assert len(self.health_info.related_pages) == 2
+        assert related_pages[1] is None
+        assert related_pages == [self_help_rp, None]
```

Comment:
I think we need an assert here that deals with the output of the command, to test that the command is giving us the correct list of broken links